tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/StorageSpec.scala

```diff
@@ -330,6 +329,79 @@ abstract class StorageSpec extends BaseSpec with CirceEq {
     }
   }
 
+  "list files" in eventually {
+    deltaClient.get[Json](s"/files/$fullId", Coyote) { (json, response) =>
+      response.status shouldEqual StatusCodes.OK
+      val mapping  = replacements(
+        Coyote,
+        "project"     -> fullId,
+        "storageId"   -> storageId,
+        "storageType" -> storageType
+      )
+      val expected = jsonContentOf("/kg/files/list.json", mapping: _*)
+      filterSearchMetadata
+        .andThen(filterResults(Set("_location")))(json) should equalIgnoreArrayOrder(expected)
+    }
+  }
```

Comment:
Is this just a test that was missing previously?

Reply:
Yes